Dockerfile

```diff
@@ -46,10 +72,10 @@ RUN source activate cudf && \
     make python_cffi && \
     make install_python
 
-ADD docs /cudf/docs
-ADD python /cudf/python
+#ADD docs /cudf/docs
+#ADD python /cudf/python
 # Needed for cudf.__version__ accuracy
-ADD .git /cudf/.git
+#ADD .git /cudf/.git
```

Comment:
Will remove these comments once we're confident new approach works

Reply:
This all worked as expected for local builds, including grabbing the version info.

Test:
```
import cudf
cudf.__version__
```
Output:
```
'0.4.0+868.g891726e.dirty'
```